### ISSUE TITLE
NoMethodError: undefined method `decrypt_table!' for

### ISSUE THREAD

**Issue by rubyconvict** [jmazzi/crypt_keeper#101]
```Service.encrypt_table!``` is fine, but ```decrypt_table!``` is undefined.
```
class Service
(...)
  crypt_keeper :options,
                encryptor: :aes_new,
                key: 'XXXXXXXXXXXXXXXXXXX',
                salt: 'XXXXXXXXXXXXXXXXXXX',
                encoding: 'UTF-8'
(...)
end

[4] pry(main)> Service.crypt_keeper_fields?
=> true
[5] pry(main)> Service.dec
Service.decorate_columns   Service.decrement_counter  Service.decrypt            
[5] pry(main)> Service.encrypt
Service.encrypt         Service.encrypt_table!  
[5] pry(main)> Service.encrypt
Service.encrypt         Service.encrypt_table!  
[5] pry(main)> Service.decrypt
Service.decrypt
[5] pry(main)> Service.decrypt_table!
NoMethodError: undefined method `decrypt_table!' for Service (call 'Service.connection' to establish a connection):Class
from /media/mkalita/storage/efiduty_local/vendor/bundle/ruby/2.2.0/gems/activerecord-4.1.6/lib/active_record/dynamic_matchers.rb:26:in `method_missing'
```

**Comment by jmazzi** [comment]
@rubyconvict which version of crypt_keeper are you using?

**Comment by rubyconvict** [comment]
0.19.0

**Comment by jmazzi** [comment]
@rubyconvict this is a master branch only feature. However, I just released `0.20.0` which includes `decrypt_table!`. Give that one a shot.

**Comment by rubyconvict** [comment]
Perfect! Thank you!
